sdcm/mgmt/cli.py

```diff
@@ -437,16 +437,7 @@ def create_backup_task(self, dc_list=None,  # pylint: disable=too-many-arguments
             cmd += f" {legacy_args}"
         res = self.sctool.run(cmd=cmd, parse_table_res=False)
         if not res.stdout:
-            raise ScyllaManagerError("Unknown failure for sctool '{}' command".format(cmd))
-
-        if res.stderr:
```

Comment:
the right thing to do is to check `if not res.ok` and then to raise an exception

Reply:
when you are right you are right. testing it manually resulted in:
```
res = mgr_cluster.manager_node.remoter.run(cmd='backup --location AWS_US_EAST_1:s3:scylla-cloud-backup-10999-11167-q71e1m')

print(res.stdout, res.stderr, res.ok)

Out[16]: <Result cmd='/opt/scylla/scylla-cloud -c /opt/scylla/scylla-dbaas.yml cluster manager sctool --cluster-id 10999 -- backup --location AWS_US_EAST_1:s3:scylla-cloud-backup-10999-11167-q71e1m' exited=0>
backup/f7883d56-9258-4cda-9478-a09ac39b819d
 {"level":"debug","ts":1627313283.6966035,"caller":"track/track.go:78","msg":"Action Start","Action":"MANAGER_COMMAND","TrackID":436651}
 True
```

Reply:
yes, whoever coded this check, based himself on some existing (and faulty IMHO) that instead of checking for the result code (or in newer remoter `res.ok`), checked for `res.stderr`... and from my experience, we could have `res.ok = True` even if we have `res.stderr`